fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -293,13 +340,29 @@ if [ ! -e "$fMRIFolder" ] ; then
   log_Msg "mkdir ${fMRIFolder}"
   mkdir "$fMRIFolder"
 fi
-cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
+${FSLDIR}/bin/imcp "$fMRIScout" "$fMRIFolder"/"$OrigScoutName"
+
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction"
+    SliceTimerCorrectionParameters=`echo ${SliceTimerCorrectionParameters} | sed 's/@/ /g'` 
+    ${FSLDIR}/bin/slicetimer -i "$fMRIFolder"/"$OrigTCSName" -o "$fMRIFolder"/"$OrigTCSName"_orig_stc -r ${TR} -v ${SliceTimerCorrectionParameters}
```

Comment:
OrigTCSName is already defined as `OrigTCSName="${NameOffMRI}_orig"`
Given that, I see no need to name the output of `slicetimer` as `"$OrigTCSName"_orig_stc`.  The `_stc` suffix alone suffices.

Reply:
Done.